CONTRIBUTING.md

```diff
@@ -9,15 +9,9 @@ code itself.
 This document contains guidelines to help you get started and how to make sure
 your contribution gets accepted, making you our newest Hydra contributor!
 
-## Communication channels
+## Ask for help
 
-Should you have any questions or need some help in getting set up, you can use
-these communication channels to reach the Hydra team and get answers in a way
-where others can benefit from it as well:
-
-- #ask-hydra on the IOG [Discord server](https://discord.gg/Qq5vNTg9PT)
-- Github [Discussions](https://github.com/input-output-hk/hydra/discussions)
-- Cardano [StackExchange](https://cardano.stackexchange.com/) using the `hydra` tag
```

Comment:
About moving the communication channels to SUPPORT.md, I don't know If you had the chance to read the corresponding commit comment but the idea is that it is a special file handled by
[github](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project)
and can benefit to people opening issues.

Reply:
About the fact that some communication channels are not only for getting help but also to suggest ideas, I'm not sure either what's the best move.

I feel like keeping the change as is for now until I can think of a better approach. I'm quite confident people will find out how to reach out to us from the link to support so that could be good enough until better.